src/recording/macromanager.cpp

```diff
@@ -73,9 +73,13 @@ void MacroManager::stopRecording() {
     // TODO(xerus) add concurrency test
     while (!m_macroRecordingState.compare_exchange_weak(armed, MacroState::Disabled))
         QThread::yieldCurrentThread();
```

Comment:
always use brackets

Reply:
Woops - you should configure clang-format to enforce this as well ;)

Reply:
Hmm, doesn't look like it has that option :/